### DISCUSSION TITLE
Temple by LexiAshe

### DISCUSSION THREAD

**Discussion by MotoLegacy** [nzp-team/nzportable#367] [Records and Speedruns]
![image](https://github.com/nzp-team/nzportable/assets/26030250/ccfe03ff-fda3-47e7-9693-92d394894a97)

Played on PSVITA because thats where my development focus has been. Got bored (and found bugs :p) so ended it on a low round 21.

**Comment by DGxShAdOwZz** [comment]
![wr2](https://github.com/nzp-team/nzportable/assets/128944577/9ca5a868-7f0b-4874-a32c-0b660813d39d)
Not a huge fan of the cqc, need to find a good place to train

**Comment by ImTander** [comment]
I just barley beat the old record, but I got it! Full video [here ](https://youtu.be/ZxRm9h98H8g) !
<img width="1920" height="1080" alt="Base Profile Screenshot 2026 03 05 - 16 11 22 03" src="https://github.com/user-attachments/assets/36ab741d-157a-4335-bfce-c6355c521cdb" />

**Comment by illorsm** [comment]
<img width="1920" height="1080" alt="image_2026-05-22_124758860" src="https://github.com/user-attachments/assets/a91bb07c-e451-40f0-8c73-22db5abe60d3" />

**Reply by M0THRAK** [reply]
Nice!

**Comment by bananajksl122** [comment]
<img width="1366" height="768" alt="fte-20260524143747-0" src="https://github.com/user-attachments/assets/115fa134-4084-466d-a6ae-f085d9efa561" />
